editor/src/clj/editor/input.clj

```diff
@@ -56,6 +59,19 @@
                        :shift (.isShiftDown scroll-event)
                        :meta (.isMetaDown scroll-event)
                        :control (.isControlDown scroll-event)))
+      :drag-over (let [drag-event ^DragEvent jfx-event]
+                   (.acceptTransferModes drag-event TransferMode/ANY)
+                   (assoc action
+                          :x (.getX drag-event)
+                          :y (.getY drag-event)))
```

Comment:
Style: `assoc` indents key-value pairs with two spaces.

Reply:
Not trying to argue, but what's the reason behind this rule? I would assume that it falls under this category https://guide.clojure.style/#vertically-align-fn-args.

Reply:
Not sure. 🤔 I think it might just be the default behavior in Cursive. There are a bunch of exceptions to that rule specified in `editor/.idea/codeStyles/Project.xml`.